slackevents/inner_events_test.go

```diff
@@ -1,7 +1,8 @@
-package slackevents
+package slackevents_test
```

Comment:
Why did you change package name?

Reply:
I see.
I think we don't need to change the package name for these tests, so let's leave it as it is 🙏

Reply:
reverted

Reply:
Thanks.